web/app.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
 use Symfony\Component\HttpFoundation\Request;
+umask(0002);
```

Comment:
When you request `app.php` on `dev` environment, it can regenerate cache. Without this line cache will be stored under `www-data:www-data` permissions, so you can not delete or regenerate it using console or build. This line change permission for generated files.